lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -1169,12 +1135,12 @@ def determine_agent_level_monitored_resource_labels(type)
 
     # Determine the common labels that should be added to all log entries
     # processed by this logging agent.
-    def determine_agent_level_common_labels
+    def get_common_labels_for_resource_type(resource_type)
```

Comment:
There are additional common labels we extract per specific resource types at entry level.

e.g. https://github.com/GoogleCloudPlatform/fluent-plugin-google-cloud/blob/master/lib/fluent/plugin/out_google_cloud.rb#L1391.

Maybe the naming should reflect that. How about something like `get_common_labels_for_group_level_resource_type`?